pages/questionThumbnail/questionThumbnail.js

```diff
@@ -0,0 +1,28 @@
+var path = require('path');
+var express = require('express');
+var fs = require('fs');
+var router = express.Router();
+
+router.get('/*', function(req, res, _next) {
```

Comment:
```suggestion
router.get('/:qid', function(req, res, _next) {
```

Use an explicit parameter name whenever possible.

Reply:
How does the '/*' parameter work? I tried your suggestion and I couldn't get it to work. I'm looking at [this page](https://expressjs.com/en/api.html#res.send) but it doesn't seem to have a clear explanation.

Reply:
See https://expressjs.com/en/guide/routing.html